doc/rados/operations/placement-groups.rst

```diff
@@ -79,16 +79,25 @@ ratio takes precedence.
 The system uses the larger of the actual ratio and the effective ratio
 for its calculation.
 
+**BIAS** is used as a multiplier to manually adjust a pool's PG based
+on prior information about how much PGs a specific pool is expected
+to have.
+
 **PG_NUM** is the current number of PGs for the pool (or the current
 number of PGs that the pool is working towards, if a ``pg_num``
 change is in progress).  **NEW PG_NUM**, if present, is what the
 system believes the pool's ``pg_num`` should be changed to.  It is
 always a power of 2, and will only be present if the "ideal" value
 varies from the current value by more than a factor of 3.
 
-The final column, **AUTOSCALE**, is the pool ``pg_autoscale_mode``,
+**AUTOSCALE**, is the pool ``pg_autoscale_mode``,
 and will be either ``on``, ``off``, or ``warn``.
 
+The final column, **PROFILE** is used to switch between two behaviors
+of the autoscaler on each pool, ``scale-up`` and ``scale-down``. 
+(See below, :ref:`_automated_scaling`)
+
+.. _automated_scaling:
```

Comment:
https://jenkins.ceph.com/job/ceph-pr-docs/51807/consoleFull#-85362248744e9240e-b50a-4693-bac0-8a991bac86ac

Reply:
Yea I guess I'll just put it without hyperlinked reference. Since the Automated Scaling section is below and not sure if we can declare it before referencing